### PR TITLE
Enable bitcode for iOS framework, rebuilt by Xcode8

### PR DESCRIPTION
**Pull request by vixentael**
Continue of https://github.com/levigroker/GRKOpenSSLFramework/pull/3

## TL;DR:
dynamic framework is re-built by Xcode8.

## FullStory:

I have prepared example projects that use frameworks built on Xcode8 (8.3.3 (8E3004b)) or Xcode9 (beta 6 (9M214v)).
https://github.com/vixentael/GRKOpenSSLFrameworkExamples

The experiment is the same: create Xcode project, integrate pre-built framework installed via pods. Simulate real app developer experience: build, archive, export (enterprise) app.

There are 4 options:

- Xcode8 project integrated with framework built by Xcode8
- Xcode8 project integrated with framework built by Xcode9
- Xcode9 project integrated with framework built by Xcode8
- Xcode9 project integrated with framework built by Xcode9

|   | X8, X8 | X8, X9 | X9, X8 | X9, X9 |
| --- | --- | --- | --- | --- |
| Build     | + | + | + | + |
| Archive | + | + | + | + |
| Export   | + | `-` | + | + |

The problem is that Xcode8 can't export app with framework built by Xcode9 due to `-disable-llvm-passes` flag error.

So I've re-built framework with Xcode8, and checked that it's working in two cases:

- App that uses framework integrated via pods
- App that uses other framework that has OpenSSL dependency linked to podspec

### CODE REVIEW

**Comment by vixentael** [comment]
@sgammon @levigroker 

Would appreciate if you guys take a look on this PR :) 
Maybe I missed smth else (I hope not)

**Comment by levigroker** [comment]
I'm trying to validate this, but developer.apple.com is currently down. :/

**Comment by levigroker** [comment]
GRKOpenSSLFramework (1.0.1.20.4) successfully published

**Comment by vixentael** [comment]
Hoorraay! 👏
